frontend/src/app/offers/create-offer/create-offer.component.html

```diff
@@ -0,0 +1,295 @@
+<div *ngIf="isAdmin; else hasOrg">
+  <p>Administrator nie może tworzyć nowych ofert.</p>
+</div>
+<ng-template #hasOrg>
+  <ng-container *ngIf="!hasOrganization; else offerForm">
+  <p>Aby dodać ofertę musisz posiadać organizację.
+    <a href="{{ djangoRoot }}/organizations/create">Możesz ją założyć tutaj</a>
+  </p>
+  </ng-container>
+</ng-template>
+<ng-template #offerForm>
+<h2>{{ inEditMode ? "Edycja" : "Tworzenie" }} oferty:</h2>
+<form #offerForm="ngForm" (ngSubmit)="onSubmit(offer)">
```

Comment:
I think for relatively complex forms reactive forms are a bit better (i.e. in terms of handling related fields or validation + cleaner HTML), but well, just an opinion :).